### DISCUSSION TITLE
pluto.tv: ads

### DISCUSSION THREAD

**Discussion by cdanapol** [uBlockOrigin/uAssets#18346] [General]
How do you use it with pluto

**Comment by MasterKia** [comment]
@cdanapol Stop opening duplicate issues and continue here:
https://github.com/uBlockOrigin/uAssets/issues/9764

**Comment by gorhill** [comment]
@cdanapol If you keep opening redundant issues like this, I will have to block you from interacting with the repo. Everybody is a volunteer here, inconsiderately burdening volunteers with pointless repeated requests is helping nobody.
